server/graphql/v2/input/NewPaymentMethodInput.js

```diff
@@ -0,0 +1,14 @@
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { NewPaymentMethodDataInput } from './NewPaymentMethodDataInput';
+
+export const NewPaymentMethodInput = new GraphQLInputObjectType({
+  name: 'NewPaymentMethodInput',
+  fields: () => ({
+    data: { type: new GraphQLNonNull(NewPaymentMethodDataInput) },
```

Comment:
It's tricky but if you're making a generic mutation, this should be optional at GraphQL level. What if the Payment Method is not a credit card?